horace_core/sqw/PixelData/@PixelData/private/binary_op_double_.m

```diff
@@ -0,0 +1,110 @@
+function obj = binary_op_double_(obj, double_array, binary_op, flip, npix)
+%% BINARY_OP_DOUBLE_ perform a binary operation between this PixelData object
+% and an array
+%
+validate_input_array(obj, double_array, npix);
+
```

Comment:
do you need to move to the first page before staring either of these loops?
`obj = obj.move_to_first_page();`